core/const.py

```diff
@@ -110,6 +112,229 @@ class ITLRegion(StrEnum):
     SouthWest = "TLK"
 
 
+class OutputCategories(StrEnum):
```

Comment:
I would be extremely wary of putting anything like this into hard-coded constants/globals in the code-base as:

1. the data is meant to be dynamic, and could theoretically change / be amended and we don't want to have to "support" that / be tied to Towns Fund specific fields in our main repo
2. Other funds will have completely different outcomes, so hard-coding is not a good solution
3. Any "custom" outputs/outcomes can be added, how are these handled in this case?